docs/managed-datahub/subscription-and-notification.md

```diff
@@ -14,6 +14,19 @@ With this feature, you can set up subscriptions to specific changes for an Entit
 This feature is especially useful in helping you stay on top of any upstream changes that could impact the assets you or your stakeholders rely on. It eliminates the need for you and your team to manually check for upstream changes, or for upstream stakeholders to identify and notify impacted users.
 As a user, you can subscribe to and receive notifications about changes such as deprecations, schema changes, changes in ownership, assertions, or incidents. You’ll always been in the know about potential data quality issues so you can proactively manage your data resources.
 
+
+## Platform Admin Notifications
+
+Datahub provides three levels of notifications: platform-level, group-level, and user-level. The group and user -level notifications are 
+described in the other sections of this doc. To set platform-level notifications (assuming you have 
+the correct permissions to do so) go to Settings > Notifications (under the Platform section - this is different than "My Notifications").
+
+The platform-level notifications are global notifications. This means if you tick the box for "An owner is added or removed from a data asset" 
+then the slack channel or email you've set up in this section will get a notification any time any asset has an owner added/removed.
+
+You'll also have the option to turn on notifications for ingestion failures (very useful to route to a platform channel to proactively ensure your
+Datahub metadata stays fresh).
```

Comment:
```
Though things like tag changes may create a lot of noisy, we recommend turning on notifications for ingestion failures. This will help you proactively ensure...
```

Or something like:
```
*Our recommendations:*
Ingestion failures: Turning global notifications on for these will...
```